pkg/subctl/cmd/join.go

```diff
@@ -218,9 +219,11 @@ func joinSubmarinerCluster(config *rest.Config, subctlData *datafile.SubctlData)
 	}
 
 	status := cli.NewStatus()
-
 	status.Start("Deploying the Submariner operator")
-	err := submarinerop.Ensure(status, config, OperatorNamespace, operatorImage)
+	err := lighthouseop.Ensure(status, config, OperatorNamespace)
+	if err != nil {
```

Comment:
It is better to remove the err check and move `submarinerop.Ensure() to after status.End() err check not needed coz status.End() already does that for us. This makes it look like submarinerop depends on lighthouseop.

Reply:
This code has been refactored.